default/scripting/ship_parts/FighterHangar/FT_HANGAR_1.focs.txt

```diff
@@ -10,6 +10,16 @@ Part
     buildtime = 1
     tags = [ "PEDIA_PC_FIGHTER_HANGAR" ]
     location = OwnedBy empire = Source.Owner
+    effectsgroups = [
+        EffectsGroup
+            scope = [[EMPIRE_OWNED_SHIP_WITH_PART(FT_BAY_1)]]
+            activation = Source
+            effects = [
+                SetCapacity partname = "FT_BAY_1" value = 4
```

Comment:
probably don't want to have this effect. would interfere with anything else that modifies the current capacity, and would make upgrade like mechanics not work.